tools/config/manage-dependencies.ts

```diff
@@ -291,11 +292,37 @@ function filterLocalRelativeImports(
         return acc;
       }, {} as LibraryWithDependencies['tsImports']);
     lib.scssImports = Object.values(lib.scssImports)
-      .filter(
-        (imp) =>
-          imp.importPath.startsWith('node_modules/') ||
-          imp.importPath.startsWith('~')
-      )
+      .filter((imp) => {
+        if (
+          imp.importPath.startsWith('.') ||
+          imp.importPath.startsWith('url(')
+        ) {
+          return false;
+        }
+        if (imp.importPath.startsWith('@')) {
+          return true;
+        }
+        // whether imports can be resolved from relative path
+        let folder = path.dirname([...imp.files][0]);
+        let file;
+        if (imp.importPath.includes('/')) {
+          const parts = imp.importPath.split('/');
+          file = parts.pop();
+          folder = folder + '/' + parts.join('/');
+        } else {
+          file = imp.importPath;
+        }
+        if (fs.existsSync(folder)) {
+          const fileList = fs.readdirSync(folder);
+          if (
+            fileList.includes(file + '.scss') ||
+            fileList.includes('_' + file + '.scss')
+          ) {
+            return false;
+          }
+        }
+        return true;
```

Comment:
Instead of creating and searching the file list of the folder we could also directly test the paths of the files that we want to know if they exist:
```
if ( fs.existSync(`${folder}/${file}.scss`)  || 
     fs.existSync(`${folder}/_${file}.scss`) ) {
      return false;
     }
```